apps/src/blockly/addons/cdoUtils.ts

```diff
@@ -297,11 +299,16 @@ export function getField(type: string) {
  * @returns {?Blockly.Field}
  */
 // Users can change their active theme using the context menu. Use this setting, if present.
-export function getUserTheme(themeOption: string | undefined) {
+export function getUserTheme(
+  themeOption: string | undefined,
+  isDefaultDarkMode: boolean = false
```

Comment:
Currently we don't have a way to change theme at all in Music Lab right? Could we just pass in some `ignoreLocalStorageTheme` param here or something for now?

Reply:
You can change theme (and toggle dark mode) using the right-click context menu in Blockly in Music Lab.